apps/storefront/src/shared/service/b2b/graphql/accountSetting.ts

```diff
@@ -21,7 +21,7 @@ const updateCustomerAccountSettings = (data: CustomFieldItems) => `mutation{
   }
 }`;
 
-const getAccountSettings = (data: CustomFieldItems) => `{
+const getAccountSettings = (data: CustomFieldItems) => `query getB2bAccountSettings {
```

Comment:
I think the convention is to use CamelCase for query names: `GetB2BAccountSettings`